samples/go/photo-dedup/cmd/photo-dedup/main.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
```

Comment:
Both jobs depend on a common code (model and dhash). It would make more sense to put (at least some of) the model code in a common location shared by all jobs working on photos, but there's no place to put it yet.